specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.0/keyvault.json

```diff
@@ -298,7 +298,7 @@
             "format": "int32",
             "minimum": 1,
             "maximum": 25,
-            "description": "Maximum number of results to return in a page. If not specified the service will return up to 25 results."
+            "description": "Optional. Specifies the maximum number of results to return in a page. If not specified, then the service uses a default of 25 as an upper bound. Setting maxresults to a value less than 1 or greater than 25 results in error response code 400 (Bad Request). If there are additional results to return, then the service returns a nextLink containing a skip token for pagination. In certain cases, the service might return fewer results than specified by maxresults (even 0 results) and also return a nextLink. Clients should not make any assumptions on the minimum number of results per page, and should enumerate all pages until the nextLink becomes null."
```

Comment:
Do we really need "optional" on all these? The fact they are not required properties (or parameters in some SDKs) already implies that; though, @johanste, is there any reason not to be explicit in docs anyway? I won't hold up the PR for this, but please acknowledge one way or another if your team wants to fix it or not.

Reply:
We do not need the Optional part in the description. Required:ness is annotated [separately from the free-form description](https://learn.microsoft.com/en-us/rest/api/keyvault/keys/get-keys/get-keys?tabs=HTTP).

Reply:
I added the `Optional` part based on how Azure Blob Storage documented their `maxresults` parameter. Sure, I'll remove it.